packages/@storefront/sayt/src/autocomplete/index.html

```diff
@@ -2,5 +2,6 @@
   <gb-sayt-past-purchases suggestions="{state.suggestions}"></gb-sayt-past-purchases>
   <gb-sayt-categories category="{state.category}" values="{state.categoryValues}"></gb-sayt-categories>
   <gb-sayt-search-terms labels="{props.labels}" suggestions="{state.suggestions}"></gb-sayt-search-terms>
+  <gb-sayt-past-search-terms past-searches="{state.pastSearches}" label="{props.labels.pastSearches}" if="{state.pastSearches.length}"></gb-sayt-past-search-terms>
```

Comment:
I would maybe position this either at the bottom here or at the bottom of `sayt` instead because my guess is that clients would want to position it with CSS either at the top or bottom of the search box